### DISCUSSION TITLE
Flashing with Baud rate 9600

### DISCUSSION THREAD

**Discussion by Quarks-1** [dzungpv/mitsubishi2MQTT#26] [Q&A]
Currently I am using [this tool](https://www.espressif.com/en/support/download/other-tools) to flash my esp2866 with the bin file in releases, however it does not let me choose a baud rate of 9600.  When I flash with 115200 baud I am able to connect to the esp8266 to set it up but when plugged into the HVAC (MLZ-KP) it does not connect to it no matter the amount of retries.  Any ideas on how to change baud rate or anything else wrong with the system?

**Comment by dzungpv** [comment]
Flash speed not relate to the program in the firmware, you can try to flash it with Arduino.
If you flash it successful, the green led will blink (for D1Mini board). If you use difference board, choose the settings for it with Arduino IDE, and choose the board you use, it is easiest way to flash the firmware.
